locmapper/Dockerfile

```
@@ -0,0 +1,2 @@
+# This Dockerfile is used to create a proxy for Github actions.
+FROM happn/locmapper
```

Since there is no version "latest" on the [DockerHub](https://hub.docker.com/r/happn/locmapper/tags?page=1&ordering=last_updated), the action does not know which version to use. 

2 solutions : precise a defined version or push a tag "latest"

```suggestion
FROM happn/locmapper:1.3
```

---

Maybe we can avoid this file and use directly this syntax in the action.yml
`image: 'docker://happn/locmapper:1.3' `

Source: https://hub.docker.com/layers/happn/locmapper/1.3/images/sha256-270d0a513b200b4a9c0c619db29f9c16e31f0219c1b4991af691a760f950b099?context=explore